src/controllers/krate/downloads.rs

```diff
@@ -67,3 +69,53 @@ pub fn downloads(req: &mut dyn Request) -> AppResult<Response> {
         meta,
     }))
 }
+
+/// Handles the `GET /crates/:crate_id/recent_downloads` route.
+pub fn recent_downloads(req: &mut dyn Request) -> AppResult<Response> {
```

Comment:
@jtgeibel any thoughts on this? it might conflict with our plans to move the historical download data to S3 🤔